pkg/util/indexeres/virtualmachine.go

```diff
@@ -0,0 +1,26 @@
+package indexeres
+
+import (
+	kubevirtv1 "kubevirt.io/api/core/v1"
+
+	"github.com/harvester/harvester/pkg/ref"
+)
+
+// The file contains the indexers which are used by controller and webhook.
+const (
+	VMByPVCIndex = "harvesterhci.io/vm-by-pvc"
+)
+
+func VMByPVC(obj *kubevirtv1.VirtualMachine) ([]string, error) {
+	results := []string{}
```

Comment:
Feel free to ignore suggestion, but I prefer nil as it doesn't heap allocate.

```suggestion
	var results []string
```